src/open_inwoner/accounts/tests/test_auth.py

```diff
@@ -964,13 +964,27 @@ def test_non_digid_user_can_edit_profile(self):
 
 
 @override_settings(ROOT_URLCONF="open_inwoner.cms.tests.urls")
-class TestRegistrationNecessary(WebTest):
+class TestRegistrationNecessary(ClearCachesMixin, WebTest):
     url = reverse_lazy("profile:registration_necessary")
 
     @classmethod
     def setUpTestData(cls):
         cms_tools.create_homepage()
 
+    def test_page_show_config_text(self):
+        config = SiteConfiguration.get_solo()
+        config.registration_text = "Hello registration text http://foo.bar/"
+        config.save()
+
+        user = UserFactory(
+            first_name="",
+            last_name="",
+            login_type=LoginTypeChoices.digid,
+        )
+        response = self.app.get(self.url, user=user)
+        self.assertContains(response, "Hello registration text")
+        self.assertContains(response, ' href="http://foo.bar/" ')
```

Comment:
I tend to follow the rule that blanket assertions against an entire content are OK when they are negative ("Assert that xyz is not in content"), but a positive assertion should target specific elements in the HTML (using pyquery or something like it). Not sure if this is right or if it's sometimes OK to assert against the entire content...

Reply:
It'd be more accurate to use the structure to test but the strings are unique enough and I don't feel like updating the whole file with playwright.